plugins/modules/zos_copy.py

```diff
@@ -2006,6 +2006,7 @@ def allocate_destination_data_set(
 
     Returns:
         bool -- True if the data set was created, False otherwise.
+        dest_params -- values of the dataset allocated if exist is and empty dictionary
```

Comment:
This return statement is confusing. I would prefer something like a dict with the attributes of the allocated destination data set if it was created by the module